Source: @pullmoll
Upstream: no
Reason: In musl libc stdin, stdout, and stderr are FILE const*

The unreadCount option may print intialization messages before the
number of unread articles. This _may_ break existing apps relying
on the output being just one number, while I know of no such app.
If someone knows how to temporarily disable stdout with musl libc
please modify this patch and move it to ../patches.

--- src/FeedReaderMain.vala	2019-07-13 21:09:05.000000000 +0200
+++ src/FeedReaderMain.vala	2020-04-14 01:26:11.436124423 +0200
@@ -98,10 +98,7 @@
 
 		if(unreadCount)
 		{
-			var old_stdout =(owned)stdout;
-			stdout = FileStream.open("/dev/null", "w");
 			Logger.init(verbose);
-			stdout =(owned)old_stdout;
 			stdout.printf("%u\n", DataBase.readOnly().get_unread_total());
 			return 0;
 		}
